Fastlane/deployment_lanes.rb

```diff
@@ -159,10 +159,12 @@
 
     branch_name = is_hotfix ? "hotfix/#{latest_release_tag}" : "release/#{latest_release_tag}"
 
-    sh "git branch #{branch_name} origin/main"
-    sh "git checkout #{branch_name}"
-
-    sh "git merge -X theirs #{latest_release_tag}" unless is_hotfix
+    unless is_hotfix
+      sh "git checkout tags/#{latest_release_tag} -b #{branch_name}"
```

Comment:
I don't think `tags/` is needed here.

```suggestion
      sh "git checkout #{latest_release_tag} -b #{branch_name}"
```